### PR TITLE
Added home address manual entry page

### PR DESCRIPTION
**Pull request by ttingle-ch**
- Added home address manual entry view
- Added unit tests for manual home address entry
- Added address validation

### CODE REVIEW

**Review by hemerem-ch — APPROVED** [review]
LGTM

File: src/types/pageURL.ts [hunk 1]
```diff
@@ -31,3 +31,9 @@ export const PERSONS_NAME = "/what-is-the-persons-name";
 export const PERSONAL_CODE = "/companies-house-personal-code";
 
 export const EMAIL_ADDRESS = "/what-is-their-email-address";
+
+export const HOME_ADDRESS = "/address-lookup";
+
+export const HOME_ADDRESS_MANUAL = "/address-manual";
+
```
Comment (sardhani-ch) [inline src/types/pageURL.ts:38]: The url should be "/enter-their-home-address" as per the ticket

File: src/routers/index.ts [hunk 2]
```diff
@@ -4,12 +4,13 @@ import {
     healthCheckController,
     indexController,
     nameController,
-    homeAddressController,
```
Comment (swalke-ch) [inline src/routers/index.ts:7]: homeAddressController should be there

Comment (swalke-ch) [inline src/routers/index.ts:7]: homeAddressController should be there

File: src/types/pageURL.ts (hunk 1 above)
Comment (ttingle-ch) [reply src/types/pageURL.ts]: Changed

**Review by swalke-ch — APPROVED** [review]

**Review by sardhani-ch — APPROVED** [review]

**Pull request closed without merging**